src/opus/utils/toc.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable prettier/prettier */
+
+// see page 15 @ https://tools.ietf.org/html/rfc6716#section-3.1
```

Comment:
Some extra information so we know what this is about without having to open the RFC page would be nice, but won't block.